config/routes.rb

```diff
@@ -7,10 +7,11 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
+  get '/doomsday', to: 'sessions#doomsday'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.